ibis/expr/types/core.py

```diff
@@ -604,6 +604,37 @@ def to_parquet(
         """
         self._find_backend(use_default=True).to_parquet(self, path, **kwargs)
 
+    @experimental
+    def to_parquet_dir(
+        self,
+        path: str | Path,
+        *,
+        params: Mapping[ir.Scalar, Any] | None = None,
+        **kwargs: Any,
+    ) -> None:
+        """Write the results of executing the given expression to a parquet file in a directory.
+
+        This method is eager and will execute the associated expression
+        immediately.
+
+        Parameters
+        ----------
+        path
+            The data source. A string or Path to the directory where the parquet file will be written.
+        filename_prefix
+            The prefix name of the parquet file. What is before `.parquet`.
+        write_batches
+            If True writes each batch into a different file.
```

Comment:
```suggestion
```

These aren't arguments in the signature, so they shouldn't be documented, or if they are supported all all backends, then we should add them to the signature with the appropriate defaults and document them.

Reply:
whoops good catch, I'll clean it up right away,